From: Roy Liu <carsomyr@gmail.com>
Date: Thu, 29 Nov 2012 16:30:23 -0500
Subject: Change the Makefile to use "install" in the place of "cp"

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 0b62581..bc40e88 100644
--- a/Makefile
+++ b/Makefile
@@ -56,9 +56,9 @@ install: 85-ebsmount.rules
 	@echo 
 
 	install -d $(PATH_BIN) $(PATH_ETC) $(PATH_INSTALL_LIB) $(PATH_UDEV_RULES)
-	cp *.conf $(PATH_ETC)
-	cp *.py $(PATH_INSTALL_LIB)
-	cp *.rules $(PATH_UDEV_RULES)
+	install -t $(PATH_ETC) -- *.conf
+	install -t $(PATH_INSTALL_LIB) -- *.py
+	install -t $(PATH_UDEV_RULES) -- *.rules
 
 	$(call with-py-executables, \
 	  ln -fs $(call libpath, $$module) $(PATH_BIN)/$(progname), \
-- 
